src/pages/workspaces/List.js

```diff
@@ -121,14 +121,22 @@ export const WorkspaceList = () => {
 
   const getWorkspace = id => _.find({ workspace: { workspaceId: id } }, workspaces)
 
-  const initialFiltered = useMemo(() => ({
-    my: _.filter(ws => !ws.public || Utils.canWrite(ws.accessLevel), workspaces),
-    public: _.filter('public', workspaces),
-    featured: _.flow(
-      _.map(featuredWs => _.find({ workspace: featuredWs }, workspaces)),
-      _.compact
-    )(featuredList)
-  }), [workspaces, featuredList])
+  const initialFiltered = useMemo(() => {
+    const [newWsList, featuredWsList] = _.partition('isNew', featuredList)
+
+    return {
+      myWorkspaces: _.filter(ws => !ws.public || Utils.canWrite(ws.accessLevel), workspaces),
+      public: _.filter('public', workspaces),
+      newAndInteresting: _.flow(
+        _.map(({ isNew, ...featuredWs }) => _.find({ workspace: featuredWs }, workspaces)),
+        _.compact
+      )(newWsList),
+      featured: _.flow(
+        _.map(featuredWs => _.find({ workspace: featuredWs }, workspaces)),
```

Comment:
We should probably explicitly `_.pick` out just the `namespace` and `name` here, so this behaves well even if there are other attributes present.